src/task.py

```diff
@@ -237,17 +237,110 @@ def get_ls_country_biome_kbas(kba_id):
                     ).keys()
                 ).map(get_ls_countries_biome_numbers)
 
+                def calc_hab_areas_state():
+                    country_code = ee.Number(country.get("isonumeric")).int()
+
+                    area = (
+                        ee.Image.pixelArea().divide(1000000).updateMask(self.watermask)
+                    )
+                    str_hab_area = area.updateMask(self.structural_habitat).rename(
+                        "str_hab_area"
+                    )
+                    habitat_image = (
+                        self.scl_image.addBands(
+                            [
+                                self.occupied_habitat_image.rename("occupied_hab_area"),
+                                str_hab_area,
+                            ]
+                        )
+                        .select(
+                            [
+                                "country",
+                                "str_hab_area",
+                                "eff_pot_hab_area",
+                                "connected_eff_pot_hab_area",
+                                "occupied_hab_area",
+                                "pa_area",
+                                "state",
+                            ]
+                        )
+                        .updateMask(self.scl_image.select("country").eq(country_code))
+                    )
+                    mode_band = ["country"]
```

Comment:
I would leave this as `mode_bands` so you can add items without having to change code